src/components/editors/backgroundEditor.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+
+import { Grid, Typography } from '@material-ui/core'
+
+import { OPTIONS } from '../../lib/options'
+
+import { Dropdown, ColorPicker, PopoverIcon } from '../SettingComponent'
+
+const BACKGROUND_EDITOR_OPTIONS = [
+  { name: 'Overlay', color: OPTIONS.backgroundColor, size: OPTIONS.backgroundSize },
+  { name: 'Window', color: OPTIONS.backgroundWindowColor, size: OPTIONS.backgroundWindowSize },
+  { name: 'Text', color: OPTIONS.backgroundTextColor, size: OPTIONS.backgroundTextSize },
+]
+
+const BackgroundEditor = () => (
+  <Grid
+    container
+    direction="column"
+    justify="flex-start"
+    alignItems="flex-start"
+    spacing={5}
+  >
+    {BACKGROUND_EDITOR_OPTIONS.map( opt => (
```

Comment:
destructure opt, because it'll be easier to understand what exactly this `opt` object provides without reading the code below